app/assets/stylesheets/_custom.scss

```diff
@@ -0,0 +1,367 @@
+	box-shadow: 0px 0px 5px 0px rgba(0,0,0,0.35);
+	margin: 40px 16px 0 16px;
+	padding:20px;
+	padding-top:10px;
+}
+
+.chart-container-bar{
+	-webkit-box-shadow: 0px 0px 5px 0px rgba(0,0,0,0.35);
+	-moz-box-shadow: 0px 0px 5px 0px rgba(0,0,0,0.35);
+	box-shadow: 0px 0px 5px 0px rgba(0,0,0,0.35);
+	margin:40px 0px 0px 1%;
+	padding:20px;
+	padding-top:10px;
+	float:left;
+	width:65%;
+	position:relative;
+	height:100%; 
+}
+
+.chart-container-pie{
+	-webkit-box-shadow: 0px 0px 5px 0px rgba(0,0,0,0.35);
+	-moz-box-shadow: 0px 0px 5px 0px rgba(0,0,0,0.35);
+	box-shadow: 0px 0px 5px 0px rgba(0,0,0,0.35);
+	margin:40px 1% 0px 0px;
+	padding:20px;
+	padding-top:10px;	
+	float:right;
+	width:31%;
+	overflow:hidden;
+	position:relative;
+	height:100%; 
+}
+
+.chart-container-barleft{
+	-webkit-box-shadow: 0px 0px 5px 0px rgba(0,0,0,0.35);
+	-moz-box-shadow: 0px 0px 5px 0px rgba(0,0,0,0.35);
+	box-shadow: 0px 0px 5px 0px rgba(0,0,0,0.35);
+	margin:40px 0px 0px 1%;
+	padding:20px;
+	padding-top:10px;
+	float:left;
+	width:49%;
+	position:relative;
+	height:100%; 
+}
+
+.chart-container-barright{
+	-webkit-box-shadow: 0px 0px 5px 0px rgba(0,0,0,0.35);
+	-moz-box-shadow: 0px 0px 5px 0px rgba(0,0,0,0.35);
+	box-shadow: 0px 0px 5px 0px rgba(0,0,0,0.35);
+	margin:40px 1% 0px 0px;
+	padding:20px;
+	padding-top:10px;	
+	float:right;
+	width:47%;
+	position:relative;
+	height:100%; 
+	overflow:auto;
+}
+
+
+.pieleft{
+	float:left;
+	width:48%;
+}
+
+.pieright{
+	float:right;
+	width:48%;
+}
+
+
+.tests_run, .pe-7{
+	margin:auto;
+	flex:100% !important;
+}
+
+
+.chart-container-clear{
+	clear:both;
+}
+
+.chart-container .pe-8 h1{
+	margin:0px;
+	margin-top:10px;
+}
+
+.chart-container-pie .chart{
+	margin-left:40px;
+}
+
+#chart_site_tests, #chart_not_responding{
+text-align:center;
+}
+
+.col{
+	padding:0px;
+}
+
+.pe-10{
+	flex:100% !important;
+}
+
+.maincol{
+	margin-left:10px;
+	margin-top:10px;
+}
+
+.maincol .side-link.btn-add {
+    margin-left: 0px;
+    margin-right:10px;
+}
+
+.maincol h1 {
+    margin:20px 20px 30px 20px;
+}
+
+.filterbar{
+height:95px;
+}
+
+.filterbar .heading{
+float:left;
+}
+
+.filterbar .topfilters{
+margin-left:250px;
+}
+
+.filterbar .topfilters .topdateto{
+float:right;
+margin-left:15px;
+}
+
+.filterbar .topfilters .topdatefrom{
+float:right;
+margin-left:30px;
+}
+
+.filterbar .topfilters .topbutton{
+float:right;
+margin-top:7px;
+text-align:right;
+margin-left:40px;
+margin-right: 20px;
+}
+
+.filterbar .topfilters .topdateperiod{
+float:right;
+margin-left:20px;
+border-right:1px solid #aaa;
+padding-right:30px;
+padding-bottom:10px;
+}
+
+.filterbar .topfilters .block{
+min-height:30px;
+line-height:30px;
+color:#000;
+}
+
+.filterbar .topfilters .Select-control,
+.filterbar .topfilters .Select-control *{
+padding-top:1px;
+padding-bottom:0px;
+}
+
+.filterbar .topfilters .Select-control .Select-arrow{
+top:5px;
+}
+
+.filterbar .topfilters .topbutton .btn-primary{
+line-height: 45px !important;
+}
+
+
```

Comment:
Please avoid multiple spacing. If you need to separate sections inside a big css you can add a comment for that. Keep in mind that this could be an indication that multiple files should be used.